lib/src/pagination/pagination.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="pagination-wrapper" [ngClass]="className" id="id">
+    <nav class="custom-pagination">
+        <ul class="pagination">
+            <ng-container *ngIf="value !== 1 && useFirstAndLast">
+                <li class="page-item" (click)="handleOnChange(1)">
+                    <button class="page-link" title="firstText">
+                        <span class="nav-action">
+                            <ng-container *ngIf="useTextNav; else angleDoubleLeftIcon">
+                                {{ firstText ? firstText : "First" }}
+                            </ng-container>
+
+                            <ng-template #angleDoubleLeftIcon>
+                                <svg name="angle-double-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
+                                    <path
+                                        d="M153.1 247.5l117.8-116c4.7-4.7 12.3-4.7 17 0l7.1 7.1c4.7 4.7 4.7 12.3 0 17L192.7 256l102.2 100.4c4.7 4.7 4.7 12.3 0 17l-7.1 7.1c-4.7 4.7-12.3 4.7-17 0L153 264.5c-4.6-4.7-4.6-12.3.1-17zm-128 17l117.8 116c4.7 4.7 12.3 4.7 17 0l7.1-7.1c4.7-4.7 4.7-12.3 0-17L64.7 256l102.2-100.4c4.7-4.7 4.7-12.3 0-17l-7.1-7.1c-4.7-4.7-12.3-4.7-17 0L25 247.5c-4.6 4.7-4.6 12.3.1 17z"
+                                    />
+                                </svg>
+                            </ng-template>
+                        </span>
+                        <span class="sr-only">{{ firstText || "First" }}</span>
+                    </button>
+                </li>
+            </ng-container>
+            <ng-container *ngIf="value !== 1">
+                <li class="page-item" (click)="handleOnChange(value - 1)">
+                    <button class="page-link" title="previousText">
```

Comment:
Don't forget the binding
```suggestion
                    <button class="page-link" [title]="previousText">
```